vanillajs/cards/app.js

```diff
@@ -0,0 +1,37 @@
+/** Creating placeholder info but will fetch this from an API or a json later**/
+const placeholderInfo = {
+  image: "https://via.placeholder.com/150/771796",
+  alt: "some placeholder text",
+  name: "Jane Doe",
+  title: "VP Engineering"
+};
+
+class Card {
+  constructor( {rootClass, dataObj, cardClass = "card"} ) {
+    this.container = document.querySelector(rootClass);
+    this.data = dataObj;
+    this.classes = {
```

Comment:
I don't understand well the purpose of this. If you're going to have just one class, I think you can just do `this.mainClass =  cardClass`